Library/Formula/libview.rb

```diff
@@ -0,0 +1,45 @@
+class Libview < Formula
+  desc "VMware's incredibly exciting widgets"
+  homepage "http://view.sourceforge.net/"
+  url "http://httpredir.debian.org/debian/pool/main/libv/libview/libview_0.6.6.orig.tar.gz"
+  sha256 "394583d2d28c334684d512eb2a260db51a877b84319f74a2425b79b5780f3ad8"
+  head "svn://svn.code.sf.net/p/view/code/trunk/libview"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gtk+"
+  depends_on "gtkmm"
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on :x11
```

Comment:
If it has a hard dependency on X11 it’ll need to be submitted to our [`homebrew/x11`](https://github.com/Homebrew/homebrew-x11) tap.